helm/idol-eductionserver/values.schema.json

```diff
@@ -96,11 +100,15 @@
                 },
                 "host": {
                     "type": "string"
+                },
+                "path": {
```

Comment:
Add servicePath (and/or rewrite/update to idol-library-example-test base schema)